packages/code-du-travail-frontend/src/outils/common/MathFormula.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { BlockMath } from "react-katex";
+import { theme } from "@socialgouv/react-ui";
+import styled from "styled-components";
+
+const MathFormula = ({ math }) => {
+  return (
+    <Wrapper>
+      <StyledFormula>
+        <BlockMath math={math} />
```

Comment:
I have no idea if it makes any sense but maybe the formula will be displayed in the snapshots if you provide math has the children instad of passing it to the math prop ?

Reply:
Cela reste identique le problème viens de l'import dynamique du composant je suis en train de mettre en place une solution avec Babel et Jest pour prendre en compte les imports dynamiques.

https://sergiodxa.com/articles/testing-in-next-dynamic-imports/